### DISCUSSION TITLE
Embed not working

### DISCUSSION THREAD

**Discussion by jpaulkeller** [discord-jda/JDA#2329] [Questions and Help] [answered]
Hello.  I've got a simple Discord/API bot, and it works fine.  I'm trying to add some embeds, but no luck so far.

I created a simple embed:

      HelpEmbed.setTitle("Title", null);
      HelpEmbed.setDescription("This is a description");
      HelpEmbed.addField("Field)", "Field Value", false);
      HelpEmbed.setColor(Color.RED);
      HelpEmbed.setFooter("Footer");

I try to send it when the help command is invoked:

   private void help(final MessageReceivedEvent event)
   {
      MessageChannel channel = event.getChannel();
      MessageEmbed embed = HelpEmbed.build();
      channel.sendMessageEmbeds(embed).queue();
   }

I can send text with sendMessage fine, but sendMessageEmbeds seems to do nothing.

Are special permissions required?  Or am I missing something else obvious?

I'm using JDA-5.0.0-alpha.11 (I've also tried 22).

Thanks for any suggestions!

**Comment by michel929** [comment]
Hello,
I'm just trying to understand the problem are you trying to create an EmbedMessage like this:

```
EmbedBuilder builder = new EmbedBuilder();

builder.setTitle("Title");
builder.setDescription("This is a description");
builder.addField("Field)", "Field Value", false);
builder.setColor(Color.RED);
builder.setFooter("Footer");

channel.sendMessageEmbeds(builder.build()).queue();
```

**Comment by jpaulkeller** [comment, marked as answer]
I fixed it, thanks.  I believe my problem was I had to enable "intents":

      builder.enableIntents(GatewayIntent.GUILD_MESSAGES, GatewayIntent.DIRECT_MESSAGES, GatewayIntent.MESSAGE_CONTENT);
